new-log-viewer/src/components/StatusBar/index.css

```diff
@@ -3,6 +3,7 @@
     bottom: 0;
 
     display: flex;
+    flex-wrap: wrap;
```

Comment:
This is to avoid the individual components in the status bar squeezing themselves when the browser window is too narrow.

One may not like that the components are wrapped outside of the viewport so that a scroll-down becomes necessary to access the wrapped components. If so, an issue can be submitted to keep track of the behaviour and we can look for a better solution in the future.